common/config-parser/src/types.rs

```diff
@@ -103,11 +103,17 @@ pub struct ConfigExecutor {
 #[derive(Clone, Debug, Deserialize)]
 pub struct ConfigRocksDB {
     pub max_open_files: i32,
+    pub cache_size:     Option<usize>,
```

Comment:
```suggestion
    #[serde(default = "default_cache_size")]
    pub cache_size:     usize,
```

I prefer to handle like this.